### PR TITLE
*: add test for #1329 and add VIEW & EVENT in builtin filter

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 

- add test for #1329
- revert VIEW support for #1105 because we choose to support sync VIEW later
- filter out EVENT by builtin regular expression, close https://github.com/pingcap/dm/issues/1342

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by lance6716** [comment]
top test cases in CI are

others: over 17m
ha_cases: 9m47
shardddl3: 8m14
shardddl2: 7m41
many_tables: 6m4
shardddl1: 5m38

other cases are less than 3 minute. Will move some test case out of `others`

File: tests/safe_mode/data/db1.increment.sql [hunk 1]
```diff
@@ -1,9 +1,11 @@
 use safe_mode_test;
 insert into t1 (uid, name) values (10003, 'Buenos Aires');
 alter table t1 add column age int;
+start transaction;
 insert into t1 (uid, name, age) values (10005, 'Buenos Aires', 200);
 insert into t2 (uid, name) values (20005, 'Aureliano José');
 insert into t1 (uid, name, age) values (10006, 'Buenos Aires', 200);
+commit;
```
Comment (GMHDBJD) [inline tests/safe_mode/data/db1.increment.sql:8]: Unfortunately, this case does not cover the wrong situation. The case can be passed even on the master branch before #1329  merged. The reason is source1 enable gtid, you can add the case for source2 or disable gtid for source1.

**Comment by lance6716** [comment]
```
[2020-12-14T02:38:06.863Z]             "msg": "[code=38032:class=dm-master:scope=internal:level=high], Message: some error occurs in dm-worker: ErrCode:36008 ErrClass:\"sync-unit\" ErrScope:\"internal\" ErrLevel:\"high\" Message:\"startLocation: [position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8635), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-29], endLocation: [position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8763), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-30]: detect inconsistent DDL sequence from source [first-location: position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8442), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-28 ddls: [ALTER TABLE `sharding_target2`.`t_target` ADD COLUMN `new_col1` INT] source: `sharding_seq`.`t2` first-location: position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8635), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-29 ddls: [ALTER TABLE `sharding_target2`.`t_target` ADD COLUMN `new_col3` INT] source: `sharding_seq`.`t2`], right DDL sequence should be [first-location: position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8056), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-26 ddls: [ALTER TABLE `sharding_target2`.`t_target` ADD COLUMN `new_col1` INT] source: `sharding_seq`.`t1` first-location: position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8249), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-27 ddls: [ALTER TABLE `sharding_target2`.`t_target` ADD COLUMN `new_col2` INT] source: `sharding_seq`.`t1` first-location: position: (test-7588bb6b-65b1-4a62-980d-3412a26e77d7-j8xbn-66jzg-bin.000001, 8635), gtid-set: 30bd0a18-3db5-11eb-9962-626e74d9f649:1-29 ddls: [ALTER TABLE `sharding_target2`.`t_target` ADD COLUMN `new_col3` INT] source: `sharding_seq`.`t2`]\" Workaround:\"Please use `show-ddl-locks` command for more details.\" , Workaround: Please execute `query-status` to check status.",
```

🤔

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**